src/Symfony/Component/Translation/Tests/Util/ArrayConverterTest.php

```diff
@@ -69,6 +69,38 @@ public static function messagesData()
                     ],
                 ],
             ],
+            [
+                // input
+                [
+                    'foo.' => 'foo.',
+                    'bar.' => 'bar.',
```

Comment:
```suggestion
                    '.bar' => '.bar',
```
As you already test trailing period in 1 word-key on the previous line

Reply:
Right.

Both cases were intended to test no affectation with or without blank spaces, but, since the spaces are no affecting the output we will simplify reading by removing those (we can remove 2 entries lines 77 and, 78).

Next update will contain this cahnges.